decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -36,6 +37,11 @@ def comments_have_votes?
       def confirmed_orders_count
         orders.finished.count
       end
+
+      # Public: Overrides the `notifiable?` Notifiable concern method.
+      def notifiable?(_context)
+        false
```

Comment:
Well, I don't think the module is really coupled to comments in any way. It's just that the only usage right now is on _every_ commentable, so I guess it's more DRY to let `Comentable` do the inclusion instead of having

```ruby
include Decidim::Notifiable
include Decidim::Comments::Commentable
```
everywhere.